packages/advanced-types/tests/types/property-helper.ts

```diff
@@ -118,6 +118,16 @@ export function testTypeEncodingAndDecoding<T extends m.Type>(
   }
 }
 
+/**
+ * This function can be used to test a type by providing the expected sequence of values:
+ *  raw: JSON ---[decoding]---> decoded: Infer<T> ---[encoding]---> encoded: JSON
+ *
+ * It's useful for testing all those types whose encode/decode process is not bijective.
+ *
+ * @param type the type to test
+ * @param validValues an array valid raw, decoded and encoded sequence
+ * @param knownInvalidValues an array of invalid values that does not pass the decode step
+ */
 export function testTypeDecodingAndEncoding<T extends m.Type>(
   type: T,
   {
```

Comment:
Why is this needed when you could call `testTypeEncodingAndDecoding` with `skipInverseCheck` set to true?